tests/data/cases/is_simple_lookup_for_doublestar_expression.py

```diff
@@ -0,0 +1,14 @@
+# flags: --preview
+m2 = None if not isinstance(dist, Normal) else m** 2 + s * 2
+m3 = None if not isinstance(dist, Normal) else m**2 + s * 2
+m4 = None if not isinstance(dist, Normal) else s**2 + s * 2
```

Comment:
This test seems the same as the previous one.

Reply:
You are right, they are testing the same thing I'll remove the duplicate test.